src/app/globals.css

```
@@ -87,6 +87,18 @@ h6 {
   font-family: var(--font-heading);
 }
 
+/* Chapter label swap in the nav (homepage journey tracker) */
+@keyframes fadeSlideIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)